logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/connection/AcceptConnectionRequestUseCase.kt

```diff
@@ -44,13 +46,15 @@ fun interface AcceptConnectionRequestUseCase {
 internal class AcceptConnectionRequestUseCaseImpl(
     private val connectionRepository: ConnectionRepository,
     private val conversationRepository: ConversationRepository,
+    private val oneOnOneResolver: OneOnOneResolver
 ) : AcceptConnectionRequestUseCase {
 
     override suspend fun invoke(userId: UserId): AcceptConnectionRequestUseCaseResult {
         return connectionRepository.updateConnectionStatus(userId, ConnectionState.ACCEPTED)
             .flatMap {
                 conversationRepository.fetchConversation(it.qualifiedConversationId)
                 conversationRepository.updateConversationModifiedDate(it.qualifiedConversationId, DateTimeUtil.currentInstant())
+                oneOnOneResolver.resolveOneOnOneConversationWithUserId(it.qualifiedToId).map { }
```

Comment:
Shouldn't this be chained after `fetchConversation`?

Something like:

```suggestion
        return connectionRepository.updateConnectionStatus(userId, ConnectionState.ACCEPTED)
            .flatMap {
                conversationRepository.fetchConversation(it.qualifiedConversationId)
                conversationRepository.updateConversationModifiedDate(it.qualifiedConversationId, DateTimeUtil.currentInstant())
            }.flatMap {
                oneOnOneResolver.resolveOneOnOneConversationWithUserId(it.qualifiedToId).map { }
            }
```
```

Reply:
Yes, it probably should be chained. I'll change it